app/views/categories/index.html.erb

```diff
@@ -1,7 +1,7 @@
 
 <div class="listbox col-xs-10 col-xs-offset-1">
   <div class="">
-    <h4><%= link_to "Add New Category", new_category_path %> <====> <span><%= link_to "See all posts", posts_path %></span></h4>
+    <h4><% if current_user.admin %><%= link_to "Add New Category", new_category_path %> <====><% end %><span><%= link_to "See all posts", posts_path %></span></h4>
```

Comment:
Can we split this across multiple lines?  I feel like `if` `end` are a lot easier to reason about on multiple line.
Also, as a random tangent the text:

```
<====>
```

That someone else put should probably be different.  Since HTML element start with the `<` character, if you want to display that character on the page you need to do `&lt;`